src/network/message.rs

```diff
@@ -202,7 +206,7 @@ impl NetworkMessage {
 
     /// Return the CommandString for the message command.
     pub fn command(&self) -> CommandString {
-        self.cmd().into()
+        CommandString::from(self.cmd()).expect("valid cmd string")
```

Comment:
This opens an attack vector to the `rust-bitcoin`-based software by just sending them invalid command string causing a crash

Reply:
I don't think so, the return value of `.cmd()` is a static string defines by the enum variant, not attacker controlled.

Reply:
I think the message in `expect` should say **invalid**. The complete message sounding like "program panicked because cmd string is valid" is strange.